### DISCUSSION TITLE
重复：临时启用或者关闭 --sub-ass-force-margins

### DISCUSSION THREAD

**Discussion by wathper** [hooke007/mpv_PlayKit#300] [回收]
sub-ass-force-margins 默认值为 no ，但是有时候需要将 ass 格式的字幕输出到视频黑边上。在 mpv.conf 反复修改该项值过于麻烦。
我希望通过一个快捷键绑定该参数以实现快速的开关，
或者是在 mpv 的控制台中输入参数直接进行开关。
我阅读了官方手册，查阅了各个网页，最终失败了。
要实现上述效果，我应该怎么做呢？

**Comment by hooke007** [comment]
input 里一堆cycle的例子
